services/response_code.proto

```diff
@@ -1398,49 +1398,9 @@ enum ResponseCodeEnum {
    */
   TOKEN_HAS_NO_ADMIN_KEY = 337;
 
-  /**
-   * The node has been marked as deleted
-   */
-  NODE_DELETED = 338;
-
-  /**
-   * A node is not found during update and delete node transaction
-   */
-  INVALID_NODE_ID = 339;
-
-  /**
-   * gossip_endpoint has a fully qualified domain name instead of ip
-   */
-  INVALID_GOSSIP_ENDPOINT = 340;
-
-  /**
-   * The node account_id is invalid
-   */
-  INVALID_NODE_ACCOUNT_ID = 341;
-
-  /**
-   * The node description is invalid
-   */
-  INVALID_NODE_DESCRIPTION = 342;
-
-  /**
-   * service_endpoint is invalid
-   */
-  INVALID_SERVICE_ENDPOINT = 343;
-
-  /**
-   * gossip_ca_certificate is invalid
-   */
-  INVALID_GOSSIP_CAE_CERTIFICATE = 344;
-
-  /**
-   * grpc_certificate_hash is invalid
-   */
-  INVALID_GRPC_CERTIFICATE = 345;
-
-  /**
-   * The maximum automatic associations value is not valid.<br/>
-   * The most common cause for this error is a value less than `-1`.
-   */
-  INVALID_MAX_AUTO_ASSOCIATIONS = 346;
+    /**
+     * The maximum automatic associations value is not valid.<br/>
+     * The most common cause for this error is a value less than `-1`.
+     */
+    INVALID_MAX_AUTO_ASSOCIATIONS = 338;
```

Comment:
I would strongly recommend we keep the number `346` here.  Changing the enumeration number for any published enum value can break a lot of things.

Reply:
I don't know if we should skip the number here. I was planning to add back rest of node code after this. INVALID_MAX_AUTO_ASSOCIATIONS is added after 0.50 release, ie. it will be in 0.51. I think we should be ok here.

Reply:
We should not issue a release with a number that we plan to change.  We'll be _changing_ this unrelated field to `338`, then possibly _changing_ back to `446` (or else we break a bunch of stuff that's building on the current `main` version by changing all the DAB numbers).

Skipping numbers is pretty common all through the current protobuf (because we have fields and enums that were removed historically without reserving the numbers), so that should be fine.

Reply:
Changed it back to 346, original value.